ShoppingList/ShoppingList/ShoppingList.csproj

```
@@ -6,4 +6,15 @@
     <UseWPF>true</UseWPF>
```

```suggestion
    <UseWPF>true</UseWPF>
    <NoWarn>AD0001</NoWarn>
```
You seem to have warnings set to be treated errors - I believe you can ignore this warning & your program should run. Alternatively, you can set treating warnings as errors to false.

---

Thank you! I wasn't sure if we were supposed to or not